mariadb/rootfs/etc/s6-overlay/s6-rc.d/mariadb-lock-post/finish

```diff
@@ -0,0 +1,21 @@
+#!/command/with-contenv bashio
+# shellcheck shell=bash
+# ==============================================================================
+# Unlock tables locked for backups
+# ==============================================================================
+
+# Due to it is started by the supervisor calling the backup_post: `/command/s6-rc -d change mariadb-lock` command,
+# it's stdout won't be redirected automatically to the add-on's log
```

Comment:
So this enables logging of the locking, nice! :+1:

Reply:
The strange thing is, that the longrun `mariadb-lock-core` service emits the log correctly, only this oneshot `mariadb-lock-post` service emits the log to the "wrong" place. Both of them are part of the same `mariadb-lock` bundle, and both of them started by the same command line (`/command/s6-rc -u change mariadb-lock`), but only the oneshot behaves this strange way.

During add-on start, any oneshot service that is part of the user bundle emits the log to the correct place, only oneshot services started manually behave this strange way.

So this smells to me, and I'm not sure this is the correct way, but maybe nobody at S6 thought these strange use-cases.

Reply:
Or maybe the supervisor, who executes these `backup_pre` and `backup_post` commands should redirect any output from these commands to the add-on's log. I can't remember whether the output (without the redirection) was part of the supervisor logs. :(

Reply:
> During add-on start, any oneshot service that is part of the user bundle emits the log to the correct place, only oneshot services started manually behave this strange way.

Hm, maybe oneshot inherit the stdout somehow from the process which initializes it. On add-on start, that is init  I guess, whereas when starting from command line, it is something else :thinking: 

Anyhow, it seems to work with this fiix :+1: 

> Or maybe the supervisor, who executes these backup_pre and backup_post commands should redirect any output from these commands to the add-on's log.

Yeah maybe we should consider that anyhow.